app/src/androidTest/java/org/fossasia/susi/ai/signup/SignUpActivityTest.kt

```diff
@@ -63,4 +64,12 @@ class SignUpActivityTest {
         onView(withId(R.id.signUp)).perform(scrollTo())
         onView(withId(R.id.signUp)).check(matches(isDisplayed()))
     }
+
+    @Test
```

Comment:
@iamareebjamal the unit test has been added here.

Reply:
Read what's a unit test and what's an instrumentation and integration test

Reply:
Are you testing UI? Then why have you written an Android Instrumentation test which is 10000 times slower than a unit test

Reply:
I wrote this 12 days ago https://github.com/fossasia/susi_android/pull/2339/files#r310375934

Seems like you don't take reviews seriously at all

Reply:
Sir, I took that. But I was asked to implement the planned actions fast, after that I should resume my task with other PR. This was said to me at the meeting. So, I started working on planned actions, before further updating this pr.

Reply:
So then why you asked for review again without changing anything?

Reply:
Sir, in the first review u asked to implement a test. I implemented the unit test. But then you said to me to implement an instrumentation test, which I added today. But, then today in the review u said to remove the instrumentation test. So, the changes have nullified now.
Unit test has already been implemented last week.

Reply:
No, I did not ask you to implement an instrumentation test. You already wrote an instrumentation test. You never wrote a unit test. 

`This is a unit test, not an instrumentation test`

Meant this is a unit test, not an instrumentation test. Why is it in instrumentation test package?

Reply:
That was a misunderstanding. I am rectifying the code. Sorry for the inconvenience caused.